src/main/kotlin/org/jetbrains/bio/big/BigWig.kt

```diff
@@ -80,24 +96,36 @@ class BigWigFile private constructor(input: RomBuffer,
             when (types[type - 1]) {
```

Comment:
https://genome.ucsc.edu/goldenpath/help/bedgraph.html "The positions listed in the input data must be in numerical order"